anndata/_torch/_dataset.py

```diff
@@ -0,0 +1,138 @@
+import scanpy as sc
+import numpy as np
+import pandas as pd
+from scipy import sparse
+from sklearn.preprocessing import LabelEncoder
+
+try:
+    import torch
+    from torch.utils.data import Dataset as DatasetTorch
+    from torch.utils.data import DataLoader
+except ImportError:
+    print('could not load torch')
```

Comment:
Why print something instead of letting the ImportError through? Also instead of printing, please use a logger:

```py
from ..logging import get_logger  # or so

logger = get_logger(__name__)
```